content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+Many automated processes exist to help you navigate the OpenShift Bugzilla workflow when you're fixing a Bugzilla bug with a
+GitHub Pull Request. Read up on the overall process [here][6] before diving into the specifics of automation.
+
+## Linking Pull Requests to Bugzilla bugs
+
+In order to declare that your pull request will fix a Bugzilla bug, all you need to do is add the bug number as a prefix to your pull
+request title, like `Bug 123: Rest of the PR title`. A PR can at most reference one Bugzilla bug.
+
+This will make the integration run a couple of validations. It will comment on GitHub with a detailled explanation of the
+validations it ran and their result, so you can easily fix issues if needed. It will also add a label that indicates the success
+of those validations to integrate with the merge automation.
+
+You can find a detailed list of the bugzilla plugin settings for your repo in the [Prow Plugins][5] page by first selecting your
+repository in the top left and then clicking on `details` of the Bugzilla plugin.
+
+## Cherry Picking/Backporting
+
+The Bugzilla automation can also assist in the backporting process by integrating with the cherry picking automation. After an
+automated cherry-pick was created for a Pull Request that referenced a Bugzilla bug, the automation will clone the original bug
+with the new target release and retitle the PR to correctly reference that new Bug. The new bug will also be marked as dependent on the original bug.
```

Comment:
nit: we should note that a cherry-pick can be done with `/cherry-pick <branch>` and will happen after a PR merges

Reply:
nit: we should note today that you have to cherrypick A to B, then B to C, in order for the bugs to get correctly chained (even though you can technically do A to C at the same time)

Reply:
nit: we should mention using bugs.ci.openshift.org when you want to declare that a bug *should* be fixed in other versions but don't have the PR(s) ready to backport yet